shared/mortgage.js

```diff
@@ -169,3 +171,9 @@ export function getMortgageTimeLeft(mortgage) {
 export function getMortgageDefaultedStatus() {
   return MORTGAGE_STATUS.defaulted
 }
+
+export function getMortgageStatus(mortgage) {
```

Comment:
Isn't this very same logic used on `ParcelMortgage.js`? Shouldn't we refactor here? (delete `getMortgageDefaultedStatus` too)

Reply:
What do you mean with `Isn't this very same logic used on ParcelMortgage.js? Shouldn't we refactor here? ` ?